qa/suites/upgrade-clients/client-upgrade-octopus-quincy/octopus-client-x/rbd/2-features/layering.yaml

```diff
@@ -0,0 +1,6 @@
+overrides:
+  ceph:
+    conf:
+      client:
+        rbd default features: 1
```

Comment:
I wonder what `2-features/defaults.yaml` and `2-features/layering.yaml` are for?  Looking at the history, they keep getting cut and pasted from previous releases but I don't think they do anything since the workunit is run with `RBD_FEATURES` environment variable set and the features from the environment variable are explicitly passed to `RBD().create` and `RBD().clone`.  It seems like the net effect is that the exact same test is run twice.

@trociny You were involved with patching `src/test/librbd/test_notify.py` in the past, does the above make sense?

Reply:
AFAICT this was the case even when `2-features` was introduced in https://github.com/ceph/ceph/pull/14638.  Despite the expected alternation between `1` and `61`, it was always `13` and later, after https://github.com/ceph/ceph/pull/18559, `61`...

Reply:
Yes, it looks like not needed and can be removed.

Reply:
Are you suggesting the line "rbd default features: 1" is to be removed?

Reply:
Remove the entire `2-features` directory and rename `3-workload` directory to `2-workload`.

Reply:
Also, please squash the fixup commits -- this should be a single commit PR.